### PR TITLE
Update CodeChain Offerings documentation

### PR DESCRIPTION
**Pull request by ScarletBlue**

### CODE REVIEW

File: docs/what-features-does-codechain-offer.rst [hunk 1]
```diff
@@ -4,9 +4,10 @@ What features does CodeChain offer?
 Key features include a built-in multi-asset management solution, which enable users to issue, transfer and manage currencies,
 tokens and/or digital items on a blockchain network, all without the need of smart contracts.
 
-CodeChain also offers programmability. More specifically, transactions are programmable. Things such as random item generation, asset evolution,
+CodeChain also offers programmability. More specifically, transactions are programmable. Asset evolution,
 fusion, etc, which are features difficult to implement with existing blockchain technologies, can now be done more efficiently
-and effortlessly.
+and effortlessly.
```
Comment (sgkim126) [inline docs/what-features-does-codechain-offer.rst]: Please remove the trailing space.

Comment (sgkim126) [reply docs/what-features-does-codechain-offer.rst]: Okay. I checked the second commit removes it.

**Comment by ScarletBlue** [comment]
I was editing on the github website, and I thought it overwrites the previous changes automatically. Does it not?

**Comment by kseo** [comment]
@sgkim126 Why don't we enable "Squash and merge" option for this repository?

**Comment by ScarletBlue** [comment]
@kseo is it possible to do that on the github site itself?

**Comment by sgkim126** [comment]
@kseo Okay. I'll enable squash merge.

**Pull request merged**